solr/solr-ref-guide/modules/query-guide/pages/common-query-parameters.adoc

```diff
@@ -400,6 +400,16 @@ If early termination is used, a `segmentTerminatedEarly` header will be included
 
 Similar to using <<timeAllowed Parameter,the `timeAllowed` Parameter>>, when early segment termination happens values such as `numFound`, xref:faceting.adoc[Facet] counts, and result xref:stats-component.adoc[Stats] may not be accurate for the entire result set.
 
+== multiThreaded Parameter
+
+[%autowidth,frame=none]
+|===
+|Optional |Default: `false`
+|===
+
+If unset or set to `false`, then the multi-threaded search implementation will not be used for the query.
+If set to `true`, then use the multi-threaded search implementation if it is supported for the query.
```

Comment:
Suggested simple wording:
> 'true' or 'false' controls if Solr may use more than one thread to satisfy the request.
> It presently is only considered for `QueryComponent` to search across Lucene's segments in parallel.

Reply:
I could see us using this parameter in the future for indexing.

Reply:
> Suggested simple wording:
> 
> > 'true' or 'false' controls if Solr may use more than one thread to satisfy the request.
> > It presently is only considered for `QueryComponent` to search across Lucene's segments in parallel.

Thanks! Will revise. I'd also intended to cross-reference the `configuring-solr-xml.adoc` documentation for `indexSearcherExecutorThreads` but lost track of that due to .adoc syntax.